book_source/01_introduction/01_project_overview.Rmd

```diff
@@ -24,18 +24,46 @@ PEcAn is not itself an ecosystem model, and it can be used to with a variety of
 
 **Acknowledgements**
 
-This material is based upon work supported by the National Science Foundation under Grant No. 1062547, 1062204, 1241894, 1261582, 1318164, 1346748, 1458021, 1638577, the National Aeronautics and Space Administration (NASA) Grant No. 13-TE13-0060, and the Energy Biosciences Institute. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation or NASA. PEcAn is a collaboration among research groups at the Department of Earth And Environment at Boston University, the Energy Biosciences Institute at the University of Illinois, the Image Spatial Data Analysis group at NCSA, the Department of Atmospheric & Oceanic Sciences at the University Wisconsin-Madison, and the Terrestrial Ecosystem Science & Technology (TEST) Group at Brookhaven National Lab.
+The PEcAn project is supported financially by the following:
+
+- National Science Foundation (NSF): 
+  - [1062547](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1062547)
+  - [1062204](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1062204)
+  - [1241894](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1241894)
+  - [1261582](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1261582)
+  - [1318164](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1318164)
+  - [1346748](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1346748)
+  - [1458021](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1458021)
+  - [1638577](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1638577)
+  - [1655095](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1655095)
+  - [1702996](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1702996)
+- National Aeronautics and Space Administration (NASA)
+  - NNX14AH65G
+  - NNX16AO13H
+  - 80NSSC17K0711
+- Energy Biosciences Institute, University of Illinois
+- Department of Earth & Environment, Boston University
```

Comment:
1. I'm pretty sure I had DOD SERDP (RC­2636) in the list, and it got lost in translation, but if not my apologies.
2. I like E&E, but I'm unaware of any occasion where they've given us a grant. @ashiklom what were you thinking of?
3. But that brings up that I did forget to include Amazon AWS and Google Summer of Code, both of which provided nontrivial in-kind financial support.

Reply:
BU E&E funded my first semester at BU (when I was on a Dean's Fellowship). But if you don't think that counts, I can take it off. I added the others in 9a2c20d4f.

Reply:
I wouldn't count that -- it was to you as a first semester grad student, not to the project

Reply:
Removed in 433ffc399